cmd/system-probe/config/config_linux_test.go

```diff
@@ -0,0 +1,42 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
if that's a linux file (by the name)
shouldn't we use `linux_bpf` build tag?

Reply:
I guess for the tracer linux with ebpf dependency is true (which would imply `linux_bpf`), but this file is strictly linux (no bpf dependency).